test/output_formatter_test.cc

```diff
@@ -224,5 +225,12 @@ TEST_F(StatidToNameTest, TestTranslations) {
   }
 }
 
+TEST_F(MediumOutputCollectorTest, FortioPedanticFormatter) {
+  const auto input_proto = loadProtoFromFile("test/test_data/output_formatter.medium.proto.gold");
+  FortioPedanticOutputFormatterImpl formatter;
+  expectEqualToGoldFile(formatter.formatProto(input_proto),
```

Comment:
This golden file is fairly long which makes it hard for the reader to figure out which portions actually are relevant to this test and which are just noise. Any way we could cut it down to the minimum required to test this new behavior?

Reply:
This makes sense, but is kind of pre-existing; I think the initial work for this just grabbed a raw arbitrary snapshot and included it for testing the origin fortio output formatter code. 
We are now more or less forced to propagate that, as our new gold file is an amended version of the original one. 
I punted this to #542 to track doing this, do that work for you?

Reply:
That works, thank you.